src/html/HtmlNodeTag.js

```diff
@@ -6,6 +6,7 @@ import { Auth, Message, PushRouteAction } from '../types';
 import styles from './HtmlStyles';
 import cascadingStyles from './cascadingStylesView';
 import cascadingStylesText from './cascadingStylesText';
+import msup from './indexBasedStyles/msup';
```

Comment:
You can just import directly like:
```js
import indexBasedStyles from './indexBasedStyles';
```
and then have `indexBasedStyles.js` file with
```js
export default {
  msup: StyleSheet.create({
  })
}
```